staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -196,6 +196,16 @@ var (
 		[]string{"verb", "resource", "subresource"},
 	)
 
+	requestFilterDuration = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name:           "apiserver_request_filter_duration_seconds",
+			Help:           "Request filter latency distribution in seconds, for each filter type",
+			Buckets:        []float64{0.0, 0.0003, 0.001, 0.003, 0.01, 0.03, 0.1, 0.3, 1.0, 5.0},
```

Comment:
Having a 0.0 bucket doesn't make much sense - we should switch that to 0.0001 or sth like that.
@tkashem